### PR TITLE
document app file structure in README.md

### PR DESCRIPTION
**Pull request by JamesKunstle**
also reference 'CONTRIBUTORS.md' in the README

Signed-off-by: James Kunstle <jkunstle@bu.edu>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,53 @@ providing an insightful and convenient interface to the Open Source project data
 
 ---
 
+## Contributing
+
+Please see our guide to contributing to this project at the following site: [CONTRIBUTORS.MD](https://github.com/oss-aspen/8Knot/blob/dev/docs/CONTRIBUTORS.md)
+
+The file-structure of our application is intuitive. Non-application files are omitted from this overview:
```
Comment (cdolfi) [inline README.md:30]: I think the file structure should be its own sections vs being under contributing

File: README.md [hunk 2]
```diff
@@ -23,6 +23,53 @@ providing an insightful and convenient interface to the Open Source project data
 
 ---
 
+## Contributing
+
+Please see our guide to contributing to this project at the following site: [CONTRIBUTORS.MD](https://github.com/oss-aspen/8Knot/blob/dev/docs/CONTRIBUTORS.md)
```
Comment (cdolfi) [inline README.md:28]: What about the other file in docs? Do you think its still relivent?

**Review by cdolfi — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (JamesKunstle) [reply README.md:28]: good point, I'll point to that too

File: README.md (hunk 1 above)
Comment (JamesKunstle) [reply README.md:30]: okay that's fair

File: README.md (hunk 2 above)
Comment (JamesKunstle) [reply README.md:28]: though we'll need to rewrite that asap

File: README.md (hunk 1 above)
Comment (JamesKunstle) [reply README.md:36]: done

**Review by cdolfi — APPROVED** [review]
lgtm

**Pull request merged**